pkg/build/nodeimage/internal/kube/builder_bazel.go

```diff
@@ -81,42 +83,27 @@ func (b *bazelBuilder) Build() (Bits, error) {
 		return nil, err
 	}
 
-	// https://docs.bazel.build/versions/master/output_directories.html
-	binDir := filepath.Join(b.kubeRoot, "bazel-bin")
-	buildDir := filepath.Join(binDir, "build")
-	bazelGoosGoarch := fmt.Sprintf("linux_%s", b.arch)
-
-	// helpers to get the binary paths which may or may not be "pure" (no cgo)
-	// Except for kubelet, these are pure in Kubernetes 1.14+
-	// kubelet is at bazel-bin/cmd/kubelet/kubelet since 1.14+
-	// TODO: do we care about building 1.13 from source once we add support
-	// for building from release binaries?
-	// https://github.com/kubernetes/kubernetes/pull/73930
-	strippedCommandPath := func(command string) string {
-		return filepath.Join(
-			binDir, "cmd", command,
-			fmt.Sprintf("%s_stripped", bazelGoosGoarch), command,
-		)
+	// This is a manual dereference of the alias created here:
+	//
+	// https://github.com/kubernetes/kubernetes/blob/b56d0acaf5bead31bd17d3b88d4a167fcbac7866/build/go.bzl#L45
+	kubeletPath, err := findGoBinary("//cmd/kubelet:_kubelet-cgo")
```

Comment:
**Question for reviewers:** How is this binary used? Is it ok to always pick the linux build target or should do a bit of extra work to deference the //cmd/kubelet alias?